dataloader_test.go

```diff
@@ -9,11 +9,12 @@ import (
 	"strconv"
 	"sync"
 	"testing"
+	"time"
 )
 
-///////////////////////////////////////////////////
-// Tests
-///////////////////////////////////////////////////
+/*
```

Comment:
apologies about this change, vim kept changing the comments to `// //////...` on save so I just gave up and changed it to this.